### PR TITLE
(#877) Remove usage of Cake.Figlet

### LINKED ISSUE #877: Remove usage of Cake.Figlet
Cake 2.0.0 now ships with Spectre.Console, which has support for Figlet text out of the box:

https://cakebuild.net/docs/writing-builds/console-output#advanced-output

As such, the Cake.Figlet addin is no longer required.

### PR DESCRIPTION
**Pull request by gep13**
Use the built in aliases that are part of Spectre.Console.

Fixes #877

### CODE REVIEW

**Comment by gep13** [comment]
This will need to be rebased once this PR has been completed: https://github.com/cake-contrib/Cake.Recipe/pull/876

**Review by augustoproiete — APPROVED** [review]
LGTM

File: Source/Cake.Recipe/Content/buildProvider.cake [hunk 1]
```diff
@@ -72,7 +72,7 @@ public enum BuildProviderType
 
 public static IBuildProvider GetBuildProvider(ICakeContext context, BuildSystem buildSystem)
 {
-    if (buildSystem.IsRunningOnAzurePipelines || buildSystem.IsRunningOnAzurePipelinesHosted)
+    if (buildSystem.IsRunningOnAzurePipelines || buildSystem.AzurePipelines.Environment.Agent.IsHosted)
```
Comment (pascalberger) [inline Source/Cake.Recipe/Content/buildProvider.cake:75]: If you want to check if you run on Azure Pipelines (On-Prem or hosted) checking for `buildSystem.IsRunningOnAzurePipelines` should be enough.

Comment (gep13) [reply Source/Cake.Recipe/Content/buildProvider.cake:75]: Sounds like a plan! Thank you!

**Pull request merged**